integration-tests/src/test/scala/play/libs/ws/ahc/AhcWSClientSpec.scala

```diff
@@ -99,10 +99,11 @@ class AhcWSClientSpec(implicit executionEnv: ExecutionEnv) extends Specification
         val responseXml = response.getBody(xml())
         responseXml.normalizeDocument()
 
-        responseXml.isEqualNode(document) must beTrue
+        responseXml.isEqualNode(document) must beTrue and {
```

Comment:
Could you please add tests to the files below instead:

1. Java: https://github.com/playframework/play-ws/blob/master/play-ahc-ws-standalone/src/test/scala/play/libs/ws/ahc/AhcWSResponseSpec.scala
2. Scala: https://github.com/playframework/play-ws/blob/master/play-ahc-ws-standalone/src/test/scala/play/api/libs/ws/ahc/AhcWSResponseSpec.scala

They are a better place to test this new method.